micmac_li3ds/xmlutil.py

```diff
@@ -12,3 +22,25 @@ def children(parent, name):
         err = 'Error: no tag "{}" in XML'.format(name)
         raise RuntimeError(err)
     return child_nodes
+
+def childFloat(parent, name):
```

Comment:
Let's follow pep8 and use the underscore notations for function names. See https://www.python.org/dev/peps/pep-0008/#id44.